CodeTests.bat

```diff
@@ -8,15 +8,24 @@ make CodeTests 2> CodeTestErrors.txt
 
 CodeTestErrors.txt
 
```

Comment:
Issue with Makefile for CodeTest Please work on along with retesting the bat file